### DISCUSSION TITLE
Setting a custom frame limit using fps values

### DISCUSSION THREAD

**Discussion by snoresx3** [ppy/osu#27032] [Ideas]
In my opinion, the frame limiter options shown are unintuitive for players who don't know much about PCs. Having the option to set a custom frame limit using a fps value would make it easier for those players, as well as to eliminate edge cases where the in-between option is best (e.g. 2x refresh rate is too low fps but 4x refresh rate has too many frame rate inconsistencies). There isn't really a way to do it on stable (apart from editing the osu!.cfg file)  or lazer, so having an intuitive way that most players are familiar with in other games would be nice.

**Comment by Rekunan** [comment]
It's been said in the dev discord that they plan on making a solution to the issue, through more fluid limiters and/or this.
